sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -136,8 +136,8 @@ trait WindowExecBase extends UnaryExecNode {
           val frame = spec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
           function match {
             case AggregateExpression(f, _, _, _, _) => collect("AGGREGATE", frame, e, f)
-            case f: FrameLessOffsetWindowFunction =>
-              collect("FRAME_LESS_OFFSET", f.fakeFrame, e, f)
+            case f: FrameLessOffsetWindowFunction => collect(
```

Comment:
for `lead(a, ignoreNulls = true)` and `lead(b, ignoreNulls = true)`, can we execute them with one window frame? I don't think so, because when `a` is null, `b` may not be null, and then the offset "cursor" of these 2 lead functions diverge.

Reply:
Maybe we can execute `lead(a, ignoreNulls = true)` and `lead(b, ignoreNulls = true)` with one window frame, but it will bring additional complexity.